pkg/flow/logging/level/level.go

```diff
@@ -0,0 +1,50 @@
+package level
```

Comment:
We're not using this `level` package anywhere yet, right? Is the intent to wire it as a substitute to the go-kit level package?

Reply:
Yep! I want to make it a drop-in replacement so we just change the import statement.

Reply:
Are we changing it in this PR, or do you want to leave it as a follow-up?

Reply:
I'll do a follow-up. The changes here also benefit the code that doesn't use the new `level` package, thanks to the log adapter checking for the level first.